packages/workbox-build/test/node/get-file-manifest-entries.js

```diff
@@ -263,38 +263,38 @@ describe('Test getFileManifestEntries', function() {
       globDirectory: path.join(__dirname, '..', '..', '..',
         'workbox-cli', 'test', 'static', 'example-project-1'),
       dynamicUrlToDependencies: {
-        '/template/url1': ['page-1.html', 'index.html'],
```

Comment:
Was this a required change or optional?

My gut says that both of these should be valid.